x-pack/platform/plugins/shared/onechat/public/application/hooks/use_conversation.ts

```diff
@@ -14,12 +14,12 @@ import { useConversationId } from './use_conversation_id';
 import { useIsSendingMessage } from './use_is_sending_message';
 import { useOnechatServices } from './use_onechat_service';
 
-const useConversation = () => {
+export const useConversation = () => {
   const conversationId = useConversationId();
   const { conversationsService } = useOnechatServices();
   const queryKey = queryKeys.conversations.byId(conversationId ?? newConversationId);
   const isSendingMessage = useIsSendingMessage();
-  const { data: conversation } = useQuery({
+  const { data: conversation, isFetched } = useQuery({
```

Comment:
Let's chat about the best way to expose these kind of properties to UI components @zacharyparikh -- this solution works now, but, I wouldn't necessarily advocate for this. We can come up with a better pattern together that suits the architecture of the codebase.